lib/http.js

```diff
@@ -355,6 +355,11 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
         // `has` query
         query['doc.indexed.attributes.name'] = {$all: has};
       }
+      if(count) {
+        const result = await storage.count({edvId, query});
+        res.json({count: result});
```

Comment:
```suggestion
        const count = await storage.count({edvId, query});
        res.json({count});
```

Reply:
one line above this we have `if(count)` which is the new `count` option from req.body you commented on below meaning if I take this suggestion we will have a naming issue that the original code does not have.

Reply:
Not sure if you will approve of this, but I did remove result and just define do

```js
res.json({count: await storage.count({edvId, query})});
return;
```
here: https://github.com/digitalbazaar/bedrock-edv-storage/pull/38/commits/becd47c30feb5b81eed50f69bad13a4cffffd749